jaba/src/pages/CreateUserPage/CreateUser.tsx

```diff
@@ -0,0 +1,204 @@
+import { useState } from 'react';
+import styles from './CreateUser.module.css';
+import './popup.css';
+import requestIconSuccess from '../../assets/requestIconSuccess.svg';
+import { addUser } from '../../backend/firestoreFunctions';
+
+const RequestAccountPage = () => {
+  const [modal, setModal] = useState(false);
+
+  const [formData, setFormData] = useState({
+    fullname: '',
+    email: '',
+    phone: '',
+    organization: '',
+    position: '',
+    address: '',
+    accountType: 'Admin',
+  });
+
+  const toggleModal = () => {
+    setModal(!modal);
+  };
+
+  const handleFormSubmit = (e: { preventDefault: () => void; }) => {
+    e.preventDefault();
+
+    if (formData.fullname.trim() !== '' 
+        && formData.email.trim() !== ''  
+        && formData.phone.trim() !== ''
+        && formData.organization.trim() !== '' 
+        && formData.position.trim() !== ''
+        && formData.address.trim() !== ''
+        && formData.accountType.trim() !== '') {
+      console.log('Form Data:', formData);
+
+      const userData = {
+        address: formData.address.trim(),
+        admin: formData.accountType === "Admin",
+        agency: formData.organization.trim(),
+        email: formData.email.trim(),
+        name: formData.fullname.trim(),
+        phone: formData.phone.trim(),
+        title: formData.position.trim()
+      }
+
+      addUser(userData)
+        .then((addedUser) => {
+          console.log('User added:', addedUser);
+          alert("Added User ");
+        })
+        .catch((error) => {
+          console.error('Error adding user:', error);
+        });
+
+      toggleModal();
+    } else {
+      alert('Please fill out all required fields.');
+    }
+  };
+
+
+  const handleInputChange = (field: string, value: string) => {
+    setFormData({
+      ...formData,
+      [field]: value,
+    });
+  };
+
+  const resetPage = () => {
+    window.location.reload();
+  };
+
+  if (modal) {
+    document.body.classList.add('active-modal');
+  } else {
+    document.body.classList.remove('active-modal');
+  }
+
+  return (
+    <div className={styles.requestPage}>
```

Comment:
Update CSS style class names to better fit this page (rename so they are different from the class names used in the PR for Task 6)